src/Deserializers/LegacyPropertyDeserializer.php

```diff
@@ -95,10 +72,23 @@ private function getPropertyId() {
 	}
 
 	/**
-	 * @return Fingerprint
+	 * @param array $serialization
+	 * @return string
```

Comment:
PHPStorm and other tools complain about incomplete documentation for a reason. Either have a valid PHPDoc section or none.